NotesApp/NotesApp.css

```diff
@@ -0,0 +1,53 @@
+* {

```

Comment:
give more meaningful and explanatory names to classes

review everywhere for this

like head does not makes any sense

Reply:
I have removed head

Reply:
I have changed the name